structure-proposal.md

```diff
@@ -0,0 +1,104 @@
+        - should reflect values commonly encounterd in applied research.
+        
+    5. choosing an appropriate software package (optional, could be standardized for enhanced evaluation)
+    6. executing the simulations
+        
+        Modularities:
+        - inclusion / exclusion of “imperfect” samples (non converged / improper solutions)
+        
+    7. file storage (to be standardized, just kept here for completionism)
+    8. troubleshooting and verification (to be standardized, just kept here for completionism)
+    9. summarizing results
+        
+        Modularities:
+        - descriptive
+        - graphical
+        - inferential
+        
+
+Here steps 1, 3, 6 and 9 correspond to Harwell et al. (1996). 2 and 4 are especially relevant for an SEM context and should thus be included explicitly. Note the step “file storage” was excluded here and should be (pre-)standardized across collaborators to enhance post-hoc evaluation. Note critically step 6 involves the decision of inclusion or exclusion of “imperfect” samples (non converged / improper solutions).
+
+Other structures identified match these steps on a rather abstract level and do not add any modularity nor do they promise increased structural standardization in simulations in the context of SEM (Law & Comas, 1991; Shannon, 1992; Smith & Marshall, 2010). Any further abstraction of the structure to fit a general concept of simulations will not benefit but rather introduce potential blur to the current stage of development of the AC approach in a SEM simulation setting. 
+
+Note. Burton et al. (2006) proposes a separate step of Criteria to evaluate the performance of statistical methods for different scenarios which is subsumed here in step 4 designing specific experimental conditions.
+
+Morris et al 2019 suggests an elaborative and detailed structure that can be aligned with the structure provided above but in a different order and notably creating a population (structural equation) model is not listed as an explicit step. The authors are rather subsuming the creation of a population model as well as the designing of specific experimental conditions under one step of “data generating mechanism”. However, this separation would benefit our AC approach in the specific context of SEM as it highlights two distinct and critical decision processes in SEM based simulations.
+
+## Conclusion
+
+Hence I propose a structure closely following Paxton et al. (2001) to create a comprehensive yet modular framework for simulation studies in the field of Structural Equation Modeling (SEM). This approach satisfies the dual imperatives of incorporating both generalizable steps from existing methodologies and unique considerations germane to SEM. By explicitly detailing these steps and identifying points of modularity, the structure creates opportunities for collaborative contributions in each critical decision-making process, thereby enhancing the study's rigor and generalizability.
+
```

Comment:
Overall thoughts:
I think our main difference is in the focus of the steps. I emphasised the seperation of different steps in Design & methods on the cost of detail in the steps of pragmatic execution. You did the opposite. However, like you partly mention, I think some of the steps you propose are really small and unnecessary, i.e. could be combined in other steps (e.g. 4,5,7,8). Especially, as your focus was on finding sensible seperation that facilitates later cooperation, I think this would simplify the collaboration process.